src/utils/unit-converter.tsx

```diff
@@ -0,0 +1,37 @@
+import { kiloUnitToUnit, unitToKiloUnit } from './rounding';
```

Comment:
missing header/copyright

Reply:
Done. I also moved the unit conversion functions to the more specific unit-converter.jsx file.